blackjax/types.py

```diff
@@ -15,3 +15,6 @@
 
 #: JAX PRNGKey
 PRNGKey = prng.PRNGKeyArray
+
+LogProbFn = Callable
```

Comment:
Is that linked to #262 ? In this case it would be better to do this in another PR that updates the entire codebase